.github/workflows/docker.yml

```diff
@@ -1,23 +1,21 @@
 name: Docker
+
 on:
   pull_request:
-    paths:
-      - .github/workflows/docker.yml
-      - Dockerfile
   push:
-    paths:
-      - .github/workflows/docker.yml
-      - Dockerfile
-    branches-ignore:
+    branches:
       - master
+  merge_group:
```

Comment:
One main difference this PR brings is that Docker CI will now be run in every PR for all supported Ubuntu versions (18.04 to 24.04), whereas previously it was only run when `Dockerfile` or the `docker.yml` workflow was changed.

The "docker" job that ran in every PR and published `homebrew/ubuntu22.04:master` on `master` pushes is now integrated into the "ubuntu (22.04)" job in the matrix below.

Reply:
Might be possible to determine the matrix dynamically to retain the original behaviour. May not be worth the effort though.

Reply:
This is fine with me 👍🏻